src/main/java/com/trailfinder/dto/ProfileDTO.java

```diff
@@ -8,12 +8,28 @@
  */
 public class ProfileDTO {
 
-	// Private Fields
```

Comment:
Didn't find this comment necessary

Reply:
Agreed... one can tell what they are by looking at the access modifier.